### PR TITLE
Only show "Shut Down Kernel" if kernel is running

### LINKED ISSUE #11392: Filebrowser Context Menu should exclude `Shut Down kernel` for documents that don't support a kernel
## Description
The context menu give the option to `Shut Down Kernel` even for documents that don't support one such as a `.zip` file and also for notebooks without a currently running kernel.

Continuation of https://github.com/jupyterlab/jupyterlab/issues/9960

## Reproduce
Open filebrowser and right click on any non-folder item

## Expected behavior
`Shut Down Kernel` only present in menu when there is a kernel to be shut down.
## Context

- Operating System and version: Linux Ubuntu 21.04 
- Browser and version: Firefox 93 
- JupyterLab version:  3.2.1

<details><summary>Troubleshoot Output</summary>
<pre>
Paste the output from running `jupyter troubleshoot` from the command line here.
You may want to sanitize the paths in the output.
</pre>
</details>

<details><summary>Command Line Output</summary>
<pre>
Paste the output from your command line running `jupyter lab` here, use `--debug` if possible.
</pre>
</details>

<details><summary>Browser Output</summary>

<pre>
Paste the output from your browser Javascript console here, if applicable.

</pre>
</details>

### PR DESCRIPTION
**Pull request by krassowski**
## References

Fixes #11392

## Code changes

Added `.jp-mod-running` as explored earlier in https://github.com/jupyterlab/jupyterlab/issues/11392#issuecomment-966861812

## User-facing changes

"Shut Down Kernel" only shows up for things with a kernel

shut-down

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by krassowski** [comment]
bot please update snapshots

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @krassowski 

Would you mind moving up a bit the fake mouse pointer in https://github.com/krassowski/jupyterlab/blob/0878902a485a4316c791c5e8baa8d90ccaeb7bd2/galata/test/documentation/general.test.ts#L217

This will keep consistent its position with the _Copy Shareable Link_ menu entry.

File: galata/test/documentation/utils.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export function generateArrow(
  * @returns The svg to inject in the page
  */
 export function positionMouse(position: { x: number; y: number }): string {
```
Comment (krassowski) [inline galata/test/documentation/utils.ts:40]: I was thinking if we should make the signature something like `position: { x: number | ElementHandle, y: number | ElementHandle }` to allow setting one of the coordinates manually while the other would be in the middle of BBox of the element. We can extract this to a new issue (along with changing the cursor to something nicer, and adding it to more screenshots).

**Comment by krassowski** [comment]
> Thanks @krassowski
> 
> Would you mind moving up a bit the fake mouse pointer in https://github.com/krassowski/jupyterlab/blob/0878902a485a4316c791c5e8baa8d90ccaeb7bd2/galata/test/documentation/general.test.ts#L217
> 
> This will keep consistent its position with the _Copy Shareable Link_ menu entry.

Thanks, done!

File: galata/test/documentation/utils.ts (hunk 1 above)
Comment (fcollonval) [reply galata/test/documentation/utils.ts:40]: Nice suggestion.

**Review by fcollonval — APPROVED** [review]
Thanks @krassowski

**Comment by fcollonval** [comment]
@meeseeksdev please backport to 3.4.x

**Pull request merged**